scripts/update/Updater.php

```diff
@@ -737,6 +737,7 @@ public function update($initialVersion)
             );
             $this->setVersion('6.4.0');
         }
+        $this->setVersion('6.4.1');
```

Comment:
Please use `$this->skip('16.4.0', '16.4.1');` in such cases.
Otherwise it will break updater scirpt

Reply:
fixed